testing/py-ostinato/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-ostinato
+_pkgname=python-ostinato
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="A python agent for the ostinato controller"
+url="http://ostinato.org/"
+arch="noarch"
+license="GPLv3"
+depends="python"
```

Comment:
According to `setup.py` included in `python-ostinato-0.7.1.tar.gz`, this module requires `protobuf>=2.3.0` — corresponding Alpine package must be specified in `depends`.